source/proxysql-conf.rst

```diff
@@ -1,5 +1,40 @@
-Configuring ProxySQL
-======================
+.. proxysql-conf::
+
+Configuring Load Balancing with ProxySQL
+========================================
+
+Percona XtraDB Cluster Operator provides choice of two cluster components to
+carry on load balancing and proxy service: you can use either `HAProxy<https://haproxy.org>`_ or `ProxySQL <https://proxysql.com/>`_.
+You can control which one to use, if any, enabling or disabling via the
+``haproxy.enabled`` and ``proxysql.enabled`` options in the ``deploy/cr.yaml``
+configuration file. 
+
+Use the following command to enable ProxySQL:
+
+.. code:: bash
+
+   kubectl patch pxc cluster1 --type=merge --patch '{ \
+    "spec": {"haproxy":{ "enabled": false }, \
+    "proxysql":{ "enabled": true } \
+    }}'
+
+.. note:: For obvious reasons the Operator will not allow the simultaneous
+   enabling of both HAProxy and ProxySQL.
+
+The resulting setup will use zero PXC node (``cluster1-pxc-0`` by default)
```

Comment:
"will use the number zero PXC node"

Reply:
also please `s/node/member/` in all docs